src/modules/launcher/Plugins/Microsoft.Plugin.Program/Programs/Win32Program.cs

```diff
@@ -240,7 +261,7 @@ public List<ContextMenuResult> ContextMenus(IPublicAPI api)
 
             var contextMenus = new List<ContextMenuResult>();
 
-            if (AppType != (uint)ApplicationTypes.INTERNET_SHORTCUT_APPLICATION)
+            if (AppType != (uint)ApplicationTypes.INTERNET_SHORTCUT_APPLICATION && AppType != (uint)ApplicationTypes.FOLDER)
```

Comment:
we might want to add generic file to this as well because a generic file shouldn't have the run as admin context menu item.

Reply:
I'll make that change. It looks like cmd, and powershell scripts can be right-click run as admin, so maybe those should also be added to the NonExeAppExtension set.

Reply:
Added script files to the executable extension set, made the hashset case insensitive and removed run as admin for generic files.